source/conf.py

```diff
@@ -15,7 +15,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Simulation Systems'
-copyright = f'{datetime.datetime.now().year},  Met Office'
+copyright = f'Met Office 2024'
```

Comment:
The first main release of this was in 2023 so a static date should probably say this